### DISCUSSION TITLE
Alarm AX Pro DISARMED

### DISCUSSION THREAD

**Discussion by sebagarayco** [petrleocompel/hikaxpro_hacs#102] [Q&A] [answered]
Sorry for creating an issue. I'm evaluating Hikvision AX Pro and want to know whether it's possible to use motion sensors even when alarm is not armed so I can use them to turn lights on and so on.

I would really appreciate your answer.

Looking forward to use this addon!

**Comment by petrleocompel** [comment, marked as answer]
There are discussion exactly for this! 
Converted to discussion...

Currently there is no possibility to use it like this. AX Pro does not have any "events" which could be used currently like this.
